crates/geometry/src/line.rs

```diff
@@ -143,9 +144,10 @@ impl<Frame, const DIMENSION: usize> Line<Frame, DIMENSION> {
     }
 }
 
-impl<From, To, const DIMENSION: usize> Mul<Line<From, DIMENSION>> for Isometry<From, To, DIMENSION>
+impl<From, To, const DIMENSION: usize, Inner> Mul<Line<From, DIMENSION>>
+    for Transform<From, To, Inner>
 where
-    Isometry<From, To, DIMENSION>: Mul<Point<From, DIMENSION>, Output = Point<To, DIMENSION>>,
+    Self: Mul<Point<From, DIMENSION>, Output = Point<To, DIMENSION>> + Copy,
```

Comment:
Isn't this the same change as in #780?

Reply:
True, I reverted this change here. Should be the same as in main now...